styles/variables-base.less

```diff
@@ -318,12 +318,17 @@
 @agate-panels-vertical-tab-margin: 0;
 @agate-panels-vertical-tab-width: inherit;
 
+// Picker
+// ---------------------------------------
 @agate-picker-border-radius: 0;
+@agate-picker-item-max-width: 300px;
+@agate-picker-item-padding: 0 24px;
```

Comment:
I'm not sure that it's fine to define silicon skin's picker item padding in `variables-base.less`.
How about defining `@agate-picker-item-padding` with `0` in `variables-base.less` and with `0 24px` in `variables-silicon.less`?